internal/services/machinelearning/machine_learning_workspace_resource_test.go

```diff
@@ -965,6 +965,7 @@ resource "azurerm_machine_learning_workspace" "test" {
 }
 
 func (r WorkspaceResource) userAssignedAndCustomManagedKey(data acceptance.TestData) string {
+	// nolint: dupword
```

Comment:
For the tests that have this, what is `dupword` flagging on?

Reply:
```
internal/services/apimanagement/api_management_backend_resource_test.go:331:21: Duplicate words (azurerm_api_management_certificate.test.thumbprint,) found (dupword)
        return fmt.Sprintf(` 
                           ^
internal/services/apimanagement/api_management_backend_resource_test.go:397:21: Duplicate words (azurerm_api_management_certificate.test.thumbprint,) found (dupword)
        return fmt.Sprintf(`
                           ^
internal/services/consumption/consumption_budget_subscription_resource_test.go:337:21: Duplicate words (contact_groups) found (dupword)
        return fmt.Sprintf(`
                           ^
internal/services/datafactory/data_factory_data_flow_resource_test.go:110:21: Duplicate words (source1) found (dupword)
        return fmt.Sprintf(`
                           ^
internal/services/datafactory/data_factory_data_flow_resource_test.go:176:21: Duplicate words (Filter1,source1) found (dupword)
        return fmt.Sprintf(`
                           ^
internal/services/datafactory/data_factory_data_flow_resource_test.go:334:21: Duplicate words (source1) found (dupword)
        return fmt.Sprintf(`
                           ^
internal/services/datafactory/data_factory_flowlet_data_flow_resource_test.go:81:21: Duplicate words (source1) found (dupword)
        return fmt.Sprintf(`
                           ^
internal/services/datafactory/data_factory_flowlet_data_flow_resource_test.go:147:21: Duplicate words (Filter1,source1) found (dupword)
        return fmt.Sprintf(`
                           ^
internal/services/machinelearning/machine_learning_workspace_resource_test.go:968:21: Duplicate words (azurerm_role_assignment.test_sa1,) found (dupword)
        return fmt.Sprintf(`

```